src/main/java/io/cdap/plugin/gcp/bigquery/util/BigQueryUtil.java

```diff
@@ -263,6 +263,10 @@ public static Schema convertFieldType(Field field, @Nullable FailureCollector co
         // bigquery has 38 digits of precision and 9 digits of scale for NUMERIC.
         // https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_types
         return Schema.decimalOf(38, 9);
+      case BIGNUMERIC:
+        // bigquery has 76 digits of precision and 38 digits of scale for BIGNUMERIC.
+        // https://cloud.google.com/bigquery/docs/reference/standard-sql/data-types#decimal_types
+        return Schema.decimalOf(77, 38);
```

Comment:
The comment says 76 and code 77.

Reply:
BTW: With the number of places this magic numbers are used, it makes sense to introduce constants for them